src/controller/CHIPDeviceController.cpp

```diff
@@ -1383,11 +1390,11 @@ CHIP_ERROR DeviceCommissioner::CloseBleConnection()
 
 void DeviceCommissioner::OnSessionEstablishmentTimeout()
 {
+    // This is called from teh session establishment timer. Please see
     VerifyOrReturn(mState == State::Initialized);
     VerifyOrReturn(mDeviceBeingCommissioned != nullptr);
 
-    CommissioneeDeviceProxy * device = mDeviceBeingCommissioned;
-    StopPairing(device->GetDeviceId());
+    StopPairing(mDeviceBeingCommissioned->GetDeviceId());
```

Comment:
I'm not sure this is right.

We seem to have a single timer, started in `Commission`.  But the way things are set up, that timer can be running in parallel with PASE establishment, and it looks like the expectation was that it will clean up both PASE establisment and commissioning?  Or something?  It's not very coherent.

We're definitely stopping this timer in some cases in PASE code, but using `mDeviceBeingCommissioned` here....

Should this clean up both `mDeviceBeingCommissioned` and `mDeviceInProcessOfPase` for now until we sort this out?

Reply:
see above - this whole timer makes no sense. It was previously intended to protect the case session, but that has its own timer now.

Reply:
Alright.  Is this timer harmless (in which case the 1.x triage for removing it makes sense)?  I'm not sure it's harmless...